src/ontology/ro-edit.owl

```diff
@@ -6571,6 +6572,18 @@ AnnotationAssertion(rdfs:label obo:RO_0017005 "contains measured amount"@en)
 ObjectPropertyDomain(obo:RO_0017005 obo:OBI_0000967)
 ObjectPropertyRange(obo:RO_0017005 obo:COB_0000121)
 
+# Object Property: obo:RO_0017007 (is contact information for)
+
+AsymmetricObjectProperty(obo:RO_0017007)
+IrreflexiveObjectProperty(obo:RO_0017007)
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0017007 "a is contact information for b, if a is a information content entity, b is a human being or an organization and a identifies a physical location or electronic resource that allows to initiate communication with a.")
+AnnotationAssertion(terms:contributor obo:RO_0017007 <https://orcid.org/0000-0003-1834-3856>)
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017007 "555-555-5555 is contact information for John Smith.")
```

Comment:
does this make sense? we typically don't see string literals appearing as the subject of our triples. What about the relation in the foaf vocabulary for phone number http://xmlns.com/foaf/spec/#term_phone? Or http://xmlns.com/foaf/spec/#term_mbox for email?

Reply:
Hi Charles, let me make sure I understand your point correctly: the example of use should specify "John Smith's phone number" is contact information for John Smith. And then the actual string literal would be connected as a value specification? Thanks for bringing this up.

Reply:
Updated the example of use to reflect Mathias's comment.